tf-deployer/deploy/deploy_helper.go

```diff
@@ -58,7 +58,7 @@ func (d *deployer) createCluster(c *clusterv1.Cluster, machines []*clusterv1.Mac
 
 	glog.Infof("Starting master creation %s", master.GetName())
 
-	if err := d.machineDeployer.Create(c, master); err != nil {
+	if err := d.machineDeployer.Create(c, master, nil); err != nil {
```

Comment:
Is a TODO needed here?

Reply:
How about an issue created that tracks the unimplemented work in tf-deployer?

Reply:
Sure. Also add a link to the issue on this thread as well. :)

Reply:
Created issue: https://github.com/kubernetes-sigs/cluster-api/issues/161